app/views/insured/consumer_roles/help_paying_coverage.html.erb

```diff
@@ -51,7 +51,7 @@
                 </div>             
               <% end %>
 
-              <% if @shop_coverage_result %>
+              <% if @shop_coverage_result == true %>
```

Comment:
Will the value here be boolean or string? Also, have to make sure this code will be compatible with review_and_submit page.